thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1027,6 +1027,7 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
     }
     inline void interChannelBarrier()
     {
+        BlockedActivityTimer t(slaveTimerStats, timeActivities);
```

Comment:
I think it is generally corrected to consider this block time, because most are being called in the context (on the stack of) start/nextRow,
but there are a couple which are in the context of stop()  [ see other comments re. not trackingstop() time ]